lib/9pfs/9pfs_vnops.c

```diff
@@ -433,7 +433,7 @@ static int uk_9pfs_read(struct vnode *vp, struct vfscore_file *fp,
 	struct uk_9pdev *dev = UK_9PFS_MD(vp->v_mount)->dev;
 	struct uk_9pfid *fid = UK_9PFS_FD(fp)->fid;
 	struct iovec *iov;
-	int rc;
+	int rc, i = 0;
```

Comment:
Since we are already fixing this, let us also fix the return variable type for `uk_9p_read/write`, which must be an `int64_t`. Since `rc` is also used with `uk_9p_open` in the write variant and there also returned, I would define a different variable `bytes` and use that in the code below. Then we have a matching variable pattern in read+write.
```suggestion
	int64_t bytes;
	int i = 0;
```
In the `<0` case you would do a `return = -(int)bytes;`

Reply:
Good idea, completely agree!